.github/workflows/mysql.yaml

```
@@ -19,9 +19,9 @@ jobs:
       DB_DATABASE: autoscaler
       DB_USER: root
       DB_PASSWORD: root
-    runs-on: ubuntu-latest
+    runs-on: ["linux", "x64"]
```

This won't work, see https://docs.github.com/en/actions/writing-workflows/choosing-where-your-workflow-runs/choosing-the-runner-for-a-job#standard-github-hosted-runners-for-public-repositories for allowed labels, if you want this workflow to be runnable.

```suggestion
    runs-on: ubuntu-latest
```